index.js

```diff
@@ -49,6 +51,24 @@ function HomeAssistantPlatform(log, config, api){
       console.log("Plugin - DidFinishLaunching");
     }.bind(this));
   }
+
+  var es = new EventSource(config.host + '/api/stream?password=' + encodeURIComponent(this.password));
```

Comment:
You're right. I was relying on the [documentation](https://home-assistant.io/developers/server_sent_events/) where the parameter is named `password`. I, personally, rely on client certificates for authentication instead of passwords so I didn't actually test this part until now.

Anyways, the browser-based [EventSource](https://developer.mozilla.org/en-US/docs/Web/API/EventSource/EventSource) doesn't seem to support additional headers, but the Node package [does](https://github.com/aslakhellesoy/eventsource/blob/master/lib/eventsource.js#L75). Which would you prefer? Query string or HTTP header?